### ISSUE TITLE
Syntax highlighting fails for `using module <relative-path>`

### ISSUE THREAD

**Issue by chriskuech** [PowerShell/EditorSyntax#191]
### System Details
Can provide if necessary

### Issue Description

Syntax highlighting does not work for relative path modules with the `using module` syntax.

#### Expected Behaviour

Syntax highlighting like this (ignore the warning)
![image](https://user-images.githubusercontent.com/3680671/70366559-05f96400-184d-11ea-9098-f892e8d233b3.png)

#### Actual Behaviour

The statement is white.
![image](https://user-images.githubusercontent.com/3680671/70366570-14478000-184d-11ea-9853-e99747a1bf54.png)

### Attached Logs

N/A

**Comment by msftrncs** [comment]
Yes, the syntax is very limited (too conservative) for the `using` keyword.   The `\` causes the entire syntaxing to fail, as its not acceptable per the grammar.  I haven't made any improvement in this area either, with my PR #155 or #156, though I have in some others such as comment based help arguments.

**Comment by chriskuech** [comment]
>The \ causes the entire syntaxing to fail, as its not acceptable per the grammar.

Ironic because due to a bug, `\` is required for relative path imports.

**Comment by msftrncs** [comment]
😄 , I don't think that is a bug, I think it is by-design as a defense in depth measure, but definitely ironic.

I might try to look in to improving the 'using' keyword's syntaxing logic.  I've done some already for the `namespace` portion in my PR's, but not for `module`.
